src/views/LoanDetails/LoanProxyDetails.js

```diff
@@ -21,6 +24,21 @@ class LoanProxyDetails extends React.Component {
         records: PropTypes.arrayOf(PropTypes.object),
       }),
     }),
+    mutator: PropTypes.shape({
+      proxy: PropTypes.shape({
```

Comment:
Could you please confirm that proxy should be isRequired?

Reply:
Fixed